source/OptimaJet.DWKit.StarterApplication/Controllers/DataController.cs

```diff
@@ -118,7 +128,23 @@ public async Task<ActionResult> ChangeData(string name, string data)
                 {
                     throw new Exception("Access denied!");
                 }
-
+                switch (name)
```

Comment:
Please add a block comment explaining why we are doing this and that we need to get this addressed with DWKit.